paasta_tools/generate_deployments_for_service.py

```diff
@@ -158,7 +164,9 @@ def get_deploy_group_mappings(soa_dir, service, old_mappings):
             )
             mapping['desired_state'] = desired_state
             mapping['force_bounce'] = force_bounce
-    return mappings
+            v2_mappings['controls'].setdefault(control_branch_alias, {})['desired_state'] = desired_state
+            v2_mappings['controls'][control_branch_alias]['force_bounce'] = force_bounce
```

Comment:
How about `service:cluster.instance`? Right now it's `service:paasta-cluster.instance`.